packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -55,28 +62,28 @@ class LightCurveViewerContainer extends Component {
   }
 
   async requestData () {
+    const { onError } = this.props
     try {
-      this.setState({ loading: asyncStates.loading })
+      // this.setState({ loading: asyncStates.loading })
```

Comment:
Should this be commented?

Reply:
It should actually be removed because the state is tracked in the store now. I just forgot to do that.